lib/primitives/tx.js

```diff
@@ -2191,7 +2169,6 @@ class TX {
 
     return {
       hash: this.txid(),
-      witnessHash: this.wtxid(),
```

Comment:
Would be good to cross check no one's relying on `tx.witnessHash`

Reply:
no one is, they depend on witnessHash method.